apps/schema/static/schema/swagger.yaml

```
@@ -2191,6 +2470,124 @@ components:
             included:
               $ref: '#/components/schemas/shipment/included'
 
+    tracking:
+      collectionResource:
+        properties:
+          type:
+            example: FeatureCollection
+
+      singularResource:
+        properties:
+          type:
+            example: Feature
+
+      pointProperties:
+        type: object
+        properties:
+          properties:
+            type: object
+            description: Metadata associated with this `Point` feature
+            allOf:
+            - $ref: '#/components/schemas/dataTypes/tracking/time'
+            - $ref: '#/components/schemas/dataTypes/tracking/uncertainty'
+            - $ref: '#/components/schemas/dataTypes/tracking/hasGPS'
+            - $ref: '#/components/schemas/dataTypes/tracking/source'
+
+      lineProperties:
+        type: object
+        properties:
+          properties:
+            type: object
+            description: Metadata associated with this `Line` feature
+            allOf:
+            - $ref: '#/components/schemas/dataTypes/tracking/linestringTimestamps'
+
+      getPointResource:
+        type: object
+        description: A GeoJSON FeatureCollection containing Point data.
+        properties:
+          data:
+            type: object
+            properties:
+              type:
+                type: string
+                description: Type of GeoJSON object
+                example: FeatureCollection
+              features:
+                type: array
+                description: GeoJSON `Feature`s in the `FeatureCollection`
+                items:
+                 $ref: '#/components/schemas/tracking/pointFeature'
+
+      pointResponse:
+        title: Point Collection
+        description: Response data with parameter as_point
+        allOf:
+        - $ref: '#/components/schemas/tracking/getPointResource'
+
+      getLineResource:
+        type: object
+        properties:
+          data:
+            type: object
+            description: A GeoJSON FeatureCollection containing Lime data.
```

:grimacing: :green_apple: 
```suggestion
            description: A GeoJSON FeatureCollection containing Line data.
```